### ISSUE TITLE
The LR downsampling method of the published dataset and validation set is different

### ISSUE THREAD

**Issue by Easoncyx** [psychopa4/PFNL#12]
Thanks for your extradionary work. 

The LR downsampling method of the published training set is **Bicubic**, the validation set is **Gaussian and sampling**. Using different method for training will lead to the training of FRVSR and VESPCN stuck when you look at the validation psnr.
